### PR TITLE
Make tracing on a scratch dataset annyoing

### LINKED ISSUE #3352: show warning for scratch datasets
see https://discuss.webknossos.org/t/annoying-background-color-for-tmpscratch-datasets/1168/4

### PR DESCRIPTION
**Pull request by daniel-wer**
Unfortunately it's not that easy to change the whole background of the tracing view due to golden layout. Instead, I've made the header look like a construction banner and show a warning. I'll ask for feedback on whether that is sufficient as well or whether they have another idea :)

I also noticed that the zoomstep-warning for volume tracings no longer worked due to golden layout (red border), which is why I removed it. We're showing an error toast anyways.

### URL of deployed dev instance (used for testing):
- https://annoyingscratch.webknossos.xyz

### Steps to test:
- Checkout the code and change the `isScratch` property of your datastore to true in the postgres datastores table. Open a tracing and be annoyed :)

### Issues:
- fixes #3352

scratch

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [x] Ready for review

### CODE REVIEW

File: app/assets/javascripts/oxalis/view/tracing_view.js [hunk 1]
```diff
@@ -30,37 +18,27 @@ const registerWebGlCrashHandler = canvas => {
...
-  handleContextMenu(event: SyntheticInputEvent<>) {
+export default function TracingView() {
+  const handleContextMenu = (event: SyntheticInputEvent<>) => {
     // hide contextmenu, while rightclicking a canvas
     event.preventDefault();
-  }
-
-  render() {
-    const divClassName = classnames({
-      "zoomstep-warning": this.props.isVolumeTracingDisallowed,
```
Comment (philippotto) [inline app/assets/javascripts/oxalis/view/tracing_view.js]: This had no effect anyway or why did you remove it?

Comment (daniel-wer) [reply app/assets/javascripts/oxalis/view/tracing_view.js]: > I also noticed that the zoomstep-warning for volume tracings no longer worked due to golden layout (red border), which is why I removed it. We're showing an error toast anyways.

Yes, it even screwed up the layout a little bit :/

**Comment by fm3** [comment]
I’d suggest going ahead with this and not waiting for the last feedback. It comes close to the original feature request, definitely closer than having nothing.
One thing: `/wKlive` is specific to one installation. Right now, they are the only one to request that feature but I’d keep it generic anyway, maybe? Something along the lines of “Datasets on this Datastore are marked as ’scratch’” should probably be enough info for the mhlab people to recognize what they have to do. What do you think?
